packages/node-cli/src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -75,20 +77,20 @@ module.exports = class ImportObjectsCommandGenerator extends BaseCommandGenerato
 		try {
 			listObjectsResult = await executeWithSpinner({
 				action: this._sdkExecutor.execute(executionContextForListObjects),
-				message: TranslationService.getMessage(MESSAGES.LOADING_OBJECTS),
+				message: NodeTranslationService.getMessage(MESSAGES.LOADING_OBJECTS),
 			});
 		} catch (error) {
-			throw TranslationService.getMessage(ERRORS.CALLING_LIST_OBJECTS, NodeUtils.lineBreak, error);
+			throw NodeTranslationService.getMessage(ERRORS.CALLING_LIST_OBJECTS, lineBreak, error);
 		}
 
 		const { data } = listObjectsResult;
 		if (SDKOperationResultUtils.hasErrors(listObjectsResult)) {
-			SDKOperationResultUtils.logResultMessage(listObjectsResult);
-			SDKOperationResultUtils.logErrors(listObjectsResult);
+			SDKOperationResultUtils.logResultMessage(listObjectsResult, this.consoleLogger);
+			listObjectsResult.errorMessages.forEach(message => this.error(message));
```

Comment:
what is `this.error(message)` ? should it be `this.consoleLogger.error(message)` ?

Reply:
Ups.. it's true! good catch